content/kubermatic/main/tutorials-howtos/kkp-configuration/custom-certificates/_index.en.md

```diff
@@ -9,10 +9,10 @@ weight = 130
 KKP 2.17+ allows to configure a "CA bundle" (a set of CA certificates) on the master cluster.
 This CA bundle is then automatically
 
-* copied into each seed cluster
-* copied into each usercluster namespace
-* copied into each usercluster (into the `kube-system` namespace)
-* used for various components, like the KKP API, machine-controller, usercluster kube-apiserver etc.
+* copied into each Seed Cluster
+* copied into each User Cluster namespace
+* copied into each User Cluster (into the `kube-system` namespace)
+* used for various components, like the KKP API, machine-controller, User Cluster kube-apiserver, etc.
```

Comment:
I'm not totally over the moon about uppercasing these terms..

Reply:
Reverted, I have no strong feelings.